packages/terra-show-hide/src/_ShowHideButton.module.scss

```diff
@@ -0,0 +1,102 @@
+@import '~terra-mixins';
+
+:local {
+  // html input[type='button'].terra-Button selector used to
+  // prevent a WebKit bug destroys native `audio` and `video` controls
+  // in Android 4. More info:
+  // https://github.com/necolas/normalize.css/issues/20
+  // https://code.google.com/p/android/issues/detail?id=34301
+  // appearance: button; set to correct the inability to style
+  // clickable types in iOS and Safari.
+  // Appearance needs to be applied only to button/input elements, as
+  // applying to anchor element will cause render issue.
+  // More info:
+  // https://github.com/necolas/normalize.css/issues/566
+  // https://github.com/necolas/normalize.css/commit/1be44f3b2a79af5d463c7ea0264fee0e6fa464ca
+  // stylelint-disable selector-no-qualifying-type
+  button.button,
+  html input[type='button'].button,
+  input[type='reset'].button,
+  input[type='submit'].button {
+    appearance: button;  // autoprefix fix into -webkit-appearance
+  }
+  // stylelint-enable selector-no-qualifying-type
+
+  .button {
+    background-color: transparent; // Remove the gray background on active links in IE 10
+    border: 0;
+    color: var(--terra-show-hide-color, #0059a8);
+    cursor: pointer;
+    display: inline-block; // prevents variant icon from breaking onto its own line
+    font-family: inherit;
+    font-size: var(--terra-show-hide-font-size, 1rem);
+    line-height: var(--terra-show-hide-line-height, 1.429);
+    outline: none;
+    padding: 0;
+    text-align: center;
+    text-decoration: var(--terra-show-hide-underline, none);
+    text-transform: none;
+    touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+    user-select: none; // Prevent text selection on buttons on mobile devices
+    white-space: normal;
+    // Remove the inner border and padding in Firefox.
+    &::-moz-focus-inner {
+      border: 0;
+      padding: 0;
+    }
+
+    &::after {
+      background-image: var(--terra-show-hide-icon, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" fill="#0059a8"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      background-repeat: no-repeat;
+      background-size: cover;
+      content: '';
+      display: inline-block;
+      height: var(--terra-show-hide-height, 0.6em);
+      margin-left: var(--terra-show-hide-margin-left, 0.4em);
+      width: var(--terra-show-hide-width, 0.6em);
```

Comment:
@gabeparra01 quick update to variable names (add "...-icon-...") and values:
```
... (--terra-show-hide-icon-height, 0.7143em);
... (--terra-show-hide-icon-margin-left, 0.21428em);
... (--terra-show-hide-icon-width, 0.7143em);
```

Reply:
https://github.com/cerner/terra-core/pull/1956/commits/7b6caaf78e2aba2caa6807234c68c674dd4aee36